protobufs/livekit_rtc.proto

```diff
@@ -353,8 +353,10 @@ message SubscriptionPermission {
 
 message SubscriptionPermissionUpdate {
   string participant_sid = 1;
-  string track_sid = 2;
+  string track_sid = 2 [deprecated = true];
```

Comment:
this is a response message, shouldn't this be in request?

Also, checking with a customer about this also. Maybe, keep `track_sid` also? I am on the fence too on that.

Reply:
haha jinx, commented above

Reply:
keeping `track_sid` seems like a footgun now that we always generate new track ids on republish

Reply:
Should the `SubscriptionPermissionUpdate` sent from the server down still translate to `track_sid` in the response?
or should we match the tracks client side also by source and/or name?

Reply:
> Should the `SubscriptionPermissionUpdate` sent from the server down still translate to `track_sid` in the response? or should we match the tracks client side also by source and/or name?

jinx again, I was adding a note about it too.